src/core/components/mod-icon/mod-icon.ts

```diff
@@ -133,7 +143,7 @@ export class CoreModIconComponent implements OnInit, OnChanges {
         this.brandedClass = this.isBranded;
 
         // No icon or local icon (not legacy), colorize it.
-        if (!this.iconUrl || this.isLocalUrl) {
+        if (!this.iconUrl || this.isLocalUrl()) {
```

Comment:
```suggestion
        if (!this.iconUrl() || this.isLocalUrl()) {
```
Shouldn't this be "this.iconUrl()"? I guess using "!this.iconUrl" will always return true because the signal instance exists, we want to check if the value is empty.